crates/storage/store_db/rocksdb.rs

```diff
@@ -1306,8 +1306,8 @@ impl StoreEngine for Store {
             hash: code_hash,
             bytecode: Bytes::copy_from_slice(bytecode),
             jump_targets: targets
-                .chunks_exact(2)
-                .map(|c| u16::from_le_bytes([c[0], c[1]]))
+                .chunks_exact(4)
+                .map(|c| u32::from_le_bytes([c[0], c[1], c[2], c[3]]))
```

Comment:
Talking with @Oppen, we think that it's better to encode the jump_target in RLP, which will be smaller than 32 bits for the common case.

Reply:
Better is a strong word for any use of RLP. But in the tradeoff IO probably wins, and RLP is better for IO than always using all the bytes.